tutorials/3d/environment_and_post_processing.rst

```diff
@@ -233,22 +353,27 @@ make characters, cars, etc. reflect on surrounding surfaces when moving around.
 
 A few user-controlled parameters are available to better tweak the technique:
 
-- **Max Steps** determines the length of the reflection. The bigger this number, the more costly it is to compute.
-- **Fade In** allows adjusting the fade-in curve, which is useful to make the contact area softer.
-- **Fade Out** allows adjusting the fade-out curve, so the step limit fades out softly.
-- **Depth Tolerance** can be used for screen-space-ray hit tolerance to gaps. The bigger the value, the more gaps will be ignored.
-- **Roughness** will apply a screen-space blur to approximate roughness in objects with this material characteristic.
+- **Max Steps:** determines the length of the reflection. The bigger this
+  number, the more costly it is to compute.
+- **Fade In:** allows adjusting the fade-in curve, which is useful to make the
+  contact area softer.
+- **Fade Out:** allows adjusting the fade-out curve, so the step limit fades out
+  softly.
+- **Depth Tolerance:** can be used for screen-space-ray hit tolerance to gaps.
+  The bigger the value, the more gaps will be ignored.
```

Comment:
```suggestion
- **Depth Tolerance:** can be used to allow screen-space-rays to pass behind objects.
  The rays will treat each object as if it has this depth in determining if it can pass behind the object.
```